static/skin/taskbar.css

```diff
@@ -44,6 +44,10 @@
     margin: 0 auto;
 }
 
+#ui_language {
```

Comment:
Also a comment for the commit message:

Typos in the title : `lanuguage`

Reply:
Done